docs/quickstart.md

```diff
@@ -160,7 +160,7 @@ You can now access the web interface by port forwarding the UI pod (mind the
 label selector) and enter `localhost:8081` in your browser:
 
 ```bash
-kubectl port-forward "$(kubectl get pod -l name=postgres-operator-ui --output='name')" 8081
+kubectl port-forward svc/postgres-operator-ui 8081:80
```

Comment:
```suggestion
kubectl port-forward svc/postgres-operator-ui 8081:8081
```

with this it works for me

Reply:
My apologies :sweat_smile:, I've updated the PR.